.circleci/config.yml

```diff
@@ -304,6 +305,36 @@ jobs:
       #      python3.8 scripts/benchmark-github-status.py --z-score-threshold 50
       - post-steps
 
+  linux-benchmarks-basic-PGO:
+    executor: build
+    environment:
+      CONBENCH_URL: "https://velox-conbench.voltrondata.run/"
+      CONBENCH_HOST: "CircleCI-runner-PGO"
```

Comment:
I've changed the hostname here even though that's not _quite_ the right slot for this in the conbench hierarchy. The reasons for doing this here is that:

* it makes it easier to find runs with PGO on the main run UI (e.g. the machine name column)
* it makes sure that benchmarks with PGO are not intermingled or compared with non-PGO benchmarks (there are other ways to do this, but those tend to be more subtle so make the first point harder...)

If this reduces oscillation, we should think about a better way to do this (probably adding PGO to the benchmark context instead of the runner name). But IMHO let's see if it helps at all before we add the extra code to put this in that place.